python/nnvm/top/nn.py

```diff
@@ -77,11 +77,21 @@ def compute_conv2d(attrs, inputs, _):
     channels = attrs.get_int("channels")
     layout = attrs["layout"]
     assert layout == "NCHW" or layout == "NHWC"
-    assert dilation == (1, 1), "not support dilate now"
+    (h, w) = dilation
+    if dilation == (1, 1):
+        inputs1 = inputs[1]
+    elif h > 1 and w > 1:
```

Comment:
if h < 1 or w < 1:
 rasie error
elif: h==1 and w==1:
 ...
else:
 ...

may be better control flow